src/main/java/com/hillel/evoApp/config/SecurityConfig.java

```diff
@@ -33,9 +33,8 @@ public AuthenticationManager authenticationManagerBean() throws Exception {
     @Override
     protected void configure(HttpSecurity http) throws Exception {
         http
-                //TODO fix a problem with CORS headers in the UI side
-                //.cors()
-                //.and()
+                .cors()
```

Comment:
Is it ready to work now?

Reply:
We will need to check it when we deploy front end part to the server.